code/lang/packs/en-US.json

```diff
@@ -39,6 +40,8 @@
     "CORPSE_INSPECT_IDENTIFIER_WEAPON": "It appears a {0} was used to kill them.",
     "CORPSE_INSPECT_IDENTIFIER_PERK": "They were carrying a {0}.",
 
+    "VOTE_NEXT_MAP": "VOTE FOR THE NEXT MAP!",
```

Comment:
I think it would make most sense using the normal writing, no uppercase. Uppercase should be handled via code imo